src/lesson3/Additionaly/ADD2.java

```diff
@@ -0,0 +1,25 @@
+package lesson3.Additionaly;
+
+import java.util.Scanner;
+
+public class ADD2 {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+
+        System.out.println("Введите сумму вклада");
+        float sum = sc.nextFloat();
+
+        float b = (sum * 7 / 100) + sum;    // число c %, за 1 месяц.
+
+        System.out.println("Введите количество месяцев");
+        int m = sc.nextInt();
+
+
+        for (int i = 1; i <= m; i++) {
+            float finish =  b * i;
```

Comment:
Is it working correct?
it seems, if the initial amount will be 100 and month will be 3, then as a result you'll have more than 300, which is incorrect